libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ReleaseHelper.java

```diff
@@ -58,6 +93,36 @@ public SubTable makeRows(Release release) throws SW360Exception {
 
     private void addFieldValueToRow(List<String> row, Release._Fields field, Release release) throws SW360Exception {
         switch (field) {
+            case COMPONENT_ID:
+                // first, add data for given field
+                row.add(fieldValueAsString(release
+                        .getFieldValue(org.eclipse.sw360.datahandler.thrift.components.Release._Fields.COMPONENT_ID)));
+
+                // second, add joined data - but only if wanted
+                // remark that headers have already been added accordingly
+                if (addAdditionalData()) {
+                    // add component type
+                    try {
+                        Component component = this.preloadedComponents.get(release.componentId);
+                        if (component != null) {
+                            row.add(fieldValueAsString(component.getFieldValue(
+                                    org.eclipse.sw360.datahandler.thrift.components.Component._Fields.COMPONENT_TYPE)));
+                        } else {
+                            row.add("");
+                        }
+                    } catch (TException e) {
```

Comment:
fieldValueAsString throws SW360Exception which was why my IDE did not show unnecessary try-catch. But without even that method, it clearly should be removed.